xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/InformationPane.java

```diff
@@ -131,4 +133,26 @@ public DocumentSyntaxPropertyPane editSyntax()
     {
         return new DocumentSyntaxPropertyPane().clickEdit();
     }
+
+    /**
+     * @return {@code true} if the information tab is found, {@code false} otherwise
+     * @since 16.4.7
+     * @since 16.10.4
+     * @since 17.1.0RC1
+     */
+    public boolean exists()
```

Comment:
This looks a bit strange because you need an instance of `InformationPane` and to get one you need to call `viewPage.openInformationDocExtraPane()`. So this forces the user to write `new InformationPage()` which is contrived. So I'd propose to make this method static instead, or move it up in the ViewPage with a generic method like: `isExtraDocPaneDisplayed(String paneName)` (or some other name). WDYT?

Reply:
Also "exists" is not the best word IMO, "displayed" seems better to me.